README.md

```diff
@@ -6,9 +6,18 @@
 - Proactively ask for feedback from teammates, graciously accept it and identify ways to act upon it.
 
 ## Role Descriptions
+
+### Individual Contributor
```

Comment:
Should we go ahead and create directories for IC and managements? Feels like that structure will be useful.